windows/src/global/delphi/chromium/Keyman.System.CEFManager.pas

```diff
@@ -128,7 +130,14 @@ procedure TCEFManager.Cleanup(Sender: TObject);
 end;
 
 destructor TCEFManager.Destroy;
+var
+  i: Integer;
 begin
+  if FWindows.Count > 0 then
+  begin
+    for i := 0 to FWindows.Count - 1 do
+      TKeymanSentryClient.Breadcrumb('trace', 'TCEFManager.Destroy:Window['+IntToStr(i)+']='+FWindows[i].GetDebugInfo);
```

Comment:
Do you need to check if `TKeymanSentryClient.Enabled`?

Reply:
That's done in the `Breadcrumb` function:

https://github.com/keymanapp/keyman/blob/b5b674fb17a3dc5b25d387cbafdf754f0f8956ed/windows/src/global/delphi/general/Keyman.System.KeymanSentryClient.pas#L120-L125